src/dbt/kipptaf/models/metrics/metricflow_time_spine.sql

```diff
@@ -0,0 +1,15 @@
+{{ config(materialized="table") }}
+
+with
+    days as (
+        {{
+            dbt_utils.date_spine(
+                datepart="day", start_date="'2000-01-01'", end_date="'2027-01-01'"
```

Comment:
@cbini just using the dates from the example, do we have official start and end dates for our time spines?